nl-core/StructureDefinitions/nl-core-LaboratoryTestResult.Specimen.xml

```diff
@@ -0,0 +1,79 @@
+    <name value="zib LaboratoryTestResult-v7.0(2024EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Specimen" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult.Specimen" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Specimen">
+      <path value="Specimen" />
+      <comment value="The Specimen concept from Dutch nl-core LaboratoryTestResult contains a concept for SpecimenMaterial and for Microorganism (isolated from the specimen material). These two concepts cannot be represented in the same Specimen resources; if there's a need to communicate both the microorganism and the sample material from which it was isolated, two instances need to be used. The instance representing the microorganism should refer in this case to the instance representing the sample material using `Specimen.parent`. All other nl-core concepts should be repeated in both resources.&#xD;&#xA;&#xD;&#xA;If the TestCode implicitly also describes a specimen (often the case if coded in LOINC), elements within the concept specimen may not conflict with it. However, if desired, these data can provide a more detailed description.This is in line with the agreements made in the IHE/Nictiz e-Lab programme.&#xD;&#xA;&#xD;&#xA;The following concepts have been moved to nl-core CollectionData (http://nictiz.nl/fhir/StructureDefinition/nl-core-CollectionData):&#xD;&#xA;* CollectedVolume (NL-CM:13.1.23)&#xD;&#xA;* CollectionPeriod (NL-CM:13.1.24)&#xD;&#xA;* CollectionDateTime NL-CM:13.1.17)&#xD;&#xA;* ReceivedDateTime (NL-CM:13.1.25)&#xD;&#xA;* CollectionMethod (NL-CM:13.1.18)" />
+      <alias value="nl-core-LaboratoryTestResult.Specimen" />
+    </element>
+    <element id="Specimen.subject">
+      <path value="Specimen.subject" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Group" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Substance" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Location" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+    </element>
+    <element id="Specimen.parent">
+      <path value="Specimen.parent" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Specimen" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-LaboratoryTestResult.Specimen" />
+      </type>
+    </element>
+    <element id="Specimen.collection.bodySite">
+      <path value="Specimen.collection.bodySite" />
+      <type>
+        <code value="CodeableConcept" />
```

Comment:
Here we should now also add the AntantomicalLocation binding to the LocationCodenlijsten with Extensible binding.